app/models/facility_district.rb

```diff
@@ -0,0 +1,37 @@
+class FacilityDistrict
+  attr_reader :name, :scope
+
+  alias_method :id, :name
+  alias_method :to_param, :name
+
+  def initialize(name:, scope: Facility.all)
```

Comment:
Should we scope by Org as well here? Only realistically an issue in Maharashtra, if there are overlapping districts between PATH and IHCI.

Reply:
I checked this, there aren't any overlapping districts. Should be okay for a quick fix.

Reply:
Yup that was my exact guess as well. Thanks for confirmation.